pryzm/chain.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pryzm",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Pryzm",
+  "chain_id": "pryzm-1",
+  "bech32_prefix": "pryzm",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
@JeremyParish69 here you go https://github.com/satoshilabs/slips/pull/1728